manifests/components/prometheus.jsonnet

```diff
@@ -394,7 +401,7 @@ local get_cm_web_hook_url = function(port, path) (
             }],
             containers_+: {
               default: kube.Container("node-exporter") {
-                image: "prom/node-exporter:v0.15.2",
+                image: "bitnami/node-exporter:v0.17.0-r1",
```

Comment:
There appear to be a large number of breaking changes since 0.15.x release (https://github.com/prometheus/node_exporter/blob/master/CHANGELOG.md). Curious to know if this upgrade is safe for BKPR?

Reply:
In our case when we upgraded from `0.14.X` to `0.16.0` we got to do some changes, but related with Grafana dashboards and the naming of the metrics. In this case it shouldn't be a problem

Reply:
indeed, nothing functionality-wise but only metrics naming fixes

Reply:
Thanks for the clarification

Reply:
jjo's earlier call for more testing was absolutely correct.  The image was "...:0.17.0-r1" (no preceding "v" on the tag), and this PR was broken in its original form.  In some ways this is a good thing, since we (as humans) should not have to scrutinise the image name/tags for basic validity and I'm kind of glad that we've trained ourselves not to perform that level of study.

On the other hand, that means our automated tests need to actually exist, and we need to learn to demand them :stuck_out_tongue: 

I have added an integration test to this PR, which I can confirm catches the original missing annotations, and (by side effect) the above incorrect node-exporter image.  It also prompted me to realise that our fluentd-es jobs were also missing annotations, so I've fixed that too.  Win.